{{cookiecutter.project_dirname}}/{{cookiecutter.django_settings_dirname}}/views.py

```diff
@@ -7,8 +7,8 @@
 class HealthView(View):
     """The health endpoint view."""
 
-    http_method_names = ("get", "options")
+    http_method_names = ("get", "head", "options")
```

Comment:
Wouldn't it be more useful to allow only a `head` request here, without the possibility of a `get`?